lex_file_name: "/home/academy/catkin_ws/src/synthts_et_ros/synthts_et/synthts_et/dct/et.dct"
lexd_file_name: "/home/academy/catkin_ws/src/synthts_et_ros/synthts_et/synthts_et/dct/et3.dct"
fn_voices: "/home/academy/catkin_ws/src/synthts_et_ros/synthts_et/synthts_et/htsvoices/eki_et_tnu.htsvoice"
output_fname: "/home/academy/.tekstHeliks/out_tnu.wav"
dur_fname: ""
speed: 1.1
half_tone: 2.0
gv_weight1: 1.0
gv_weight2: 1.2
print_label: false
print_utt: false
write_raw: false
write_durlabel: false